src/library/simplified/extension/number-of-guests.js

```diff
@@ -3,6 +3,16 @@
     {
         b['numberOfGuests'] = b['entity.hotels.search.hotelParameters.numberOfGuests'];
     }
+    if (utag.isRailSearchResults() && b['entity.railSearch.searchResults.railLegs.0'] != undefined && b['entity.railSearch.searchResults.railLegs.0.railOfferItems.0'] != undefined && 
+	b['entity.railSearch.searchResults.railLegs.0.railOfferItems.0.travelers.0'] != undefined) 
+    {
+        var numberOfGuests = 0;
+        for (var guest = 0 ; guest < b.entity.railSearch.searchResults.railLegs.get(0).railOfferItems.get(0).travelers.length; guest++) 
```

Comment:
Moving forward, we should avoid calculating the number of guests for all LOBs. I believe the UDO has and object where we can expose the total number of guests/travelers that we can use for each LOB. Currently, there is a bug where we are not calculating it correctly. Specifically for flights when using child in-lap.